.travis.yml

```diff
@@ -8,7 +8,7 @@ notifications:
     email: false
 script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.checkout("InterestRates"); Pkg.add("Coverage"); Pkg.test("InterestRates"; coverage=true)'
```

Comment:
If we remove `Pkg.checkout("InterestRates")`, will travis run the build and tests for master branch, or the lastest version registered on metadata?